src/plugins/console/common/constants/autocomplete_definitions.ts

```diff
@@ -16,3 +16,6 @@ export const AUTOCOMPLETE_DEFINITIONS_FOLDER = resolve(
 export const GENERATED_SUBFOLDER = 'generated';
 export const OVERRIDES_SUBFOLDER = 'overrides';
 export const MANUAL_SUBFOLDER = 'manual';
+
+export const API_REFERENCE_DOC_LINK =
+  'https://www.elastic.co/guide/en/starting-with-the-elasticsearch-platform-and-its-solutions/current/api-reference.html';
```

Comment:
@florent-leborgne Do we need to take into account the users language preference when constructing this url? that is if the user has es-ES the link should instead be `https://www.elastic.co/guide/es/starting...html`

Reply:
Good to know!